cloudtests.py

```diff
@@ -64,8 +64,12 @@ class TestJournalWritten(unittest.TestCase):
     def test_journal_written(self):
         """Test to check if journal gets written to disk"""
 
-        out, err, eid = system("sudo ls -l /proc/$(pgrep journal)/fd/ | grep journal")
+        out, err, eid = system("systemctl show systemd-journald.service -p MainPID")
         out = out.decode('utf-8')
+        pid = out[8:-1]
```

Comment:
Since the command returns `b'MainPID=500\n'` so slicing this way makes it return the `PID` only. :)